AudioLinkUnityProject/Packages/.gitignore

```diff
@@ -1,2 +1,3 @@
 com.vrchat.*/
 !com.vrchat.core.*/
+/*/
```

Comment:
wtf are you doing

Reply:
I cherrypicked the resolver update commit onto this branch because I needed/wanted to use newer U# and not have to keep manually swapping to it everytime I switched to this branch

Reply:
what does that have to do with the gitignore? This doesn't seem like something that should be pushed...
btw, you can ignore files locally. might wanna look into that